persist/sqlite/consensus.go

```diff
@@ -1147,14 +1147,14 @@ func addHosts(tx *txn, hosts []explorer.Host) error {
 	return nil
 }
 
-func addHostScans(tx *txn, scans []explorer.HostScan) error {
-	unsuccessfulStmt, err := tx.Prepare(`UPDATE host_info SET last_scan = ?, last_scan_successful = 0, total_scans = total_scans + 1, failed_interactions = failed_interactions + 1 WHERE public_key = ?`)
+func addHostScans(tx *txn, backoffInterval time.Duration, scans []explorer.HostScan) error {
```

Comment:
More of a nit, but `NextScan time.Time` could be added to the `HostScan` instead of calculating it in the database. Saves a parameter and cleans up the logic here.

Reply:
If we wanted to do the calculation outside the query we'd need to retrieve the number of previous failures (`failed_interactions_streak`) for that host from the DB first which I feel would complicate things too much to be worth it

Reply:
Does it complicate things because you feel the implementation is complicated or because it's outside the scope of this PR? How would you go about implementing this?

Reply:
Actually, we could retrieve the `failed_interactions_streak` count alongside the other details about a host in HostsForScanning.  Then it'd be pretty easy.